src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationEngine.java

```diff
@@ -445,7 +445,17 @@ else if (kruizeObject.getMode().equalsIgnoreCase(AnalyzerConstants.AUTO) || krui
                         experimentName, interval_start_time, interval_end_time));
                 kruizeObject.setValidation_data(new ValidationOutputData(false, e.getMessage(), HttpServletResponse.SC_INTERNAL_SERVER_ERROR));
             }
-        } catch (Exception | FetchMetricsError | InvalidModelException | InvalidTermException e) {
+        } catch (FetchMetricsError e) {
```

Comment:
Looks like you're setting the same error message in all the catch blocks. If it's expected, you can remove multiple catch blocks and just put one.

Reply:
converted to a single catch block